### PR TITLE
Paper__Halisa Saipulla

### PR DESCRIPTION
**Pull request by lisa1501**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <section class="city__section">
+            <h2>City Name</h2>
+            <input type="text" id="cityNameInput" value="San Francisco">
+            <button id="cityNameReset" class="city-name-reset-btn">Reset</button>
+        </section>
+
+        <section class="temperature__section">
+            <h2>Temperature</h2>
+            <div id = "container">            
+                <h1 id="temp" class="orange">71</h1>
```
Comment (CheezItMan) [inline index.html:34]: Your initial class here is `orange` but 71 is a `green` color.

```suggestion
                <h1 id="temp" class="green">71</h1>
```

File: scripts/index.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    state.temp +=1;
+    tempContainer.textContent=`${state.temp}`;
+    
+    updateColorofTemp(state.temp);
+    updateGarden(state.temp);
+};
+
+const decreatTemp = ()=>{
+    const tempContainer=document.getElementById("temp")
+    state.temp +=1;
```
Comment (CheezItMan) [inline scripts/index.js]: Ooops!

```suggestion
    state.temp -=1;
```

File: scripts/index.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    tempContainer.textContent=`${state.temp}`;
+
+    updateColorofTemp(state.temp);
+    updateGarden(state.temp);
+};
+
+const updateColorofTemp=()=>{
```
Comment (CheezItMan) [inline scripts/index.js:18]: Nothing here handles if the temp is 49 or below.

File: scripts/index.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        state.color="green";
+        
+    }else if(state.temp>=60){
+        state.color="orange";
+        
+    }
+    tempContainer.classList=state.color;
+};
+
+const updateGarden=()=>{
```
Comment (CheezItMan) [inline scripts/index.js:38]: What about temps of 59 or below?

File: scripts/index.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+};
+
+const increatTemp = ()=>{
+    const tempContainer=document.getElementById("temp")
+    state.temp +=1;
+    tempContainer.textContent=`${state.temp}`;
+    
+    updateColorofTemp(state.temp);
+    updateGarden(state.temp);
+};
```
Comment (CheezItMan) [inline scripts/index.js]: I just want to note that you could combine `increatTemp` and `decreatTemp` by adding a parameter

```suggestion
const changeTemp = (increment)=>{
    const tempContainer=document.getElementById("temp")
    state.temp +=increment;
    tempContainer.textContent=`${state.temp}`;
    
    updateColorofTemp(state.temp);
    updateGarden(state.temp);
};
```

Then you can register the event handler with an anonymous function like:

```javascript
increaseTempValue.addEventListener("click", () => changeTemp( +1 ) );
decreaseTempValue.addEventListener("click", () => changeTemp( -1 ) );

```

**Review by CheezItMan — COMMENTED** [review]
Halisa, overall this is pretty good, but you missed some key details.  You demonstrated you understand how to modify the DOM in JS, but some of the missing details make the project seem rushed.   Take a look at my feedback and let me know if you have questions.

File: scripts/index.js (hunk 2 above)
Comment (lisa1501) [reply scripts/index.js]: Thank you for your feedback, Chris ! I have refactored my code follow your commits . Really like the combination func of  increatTemp and decreatTemp.

**Pull request closed without merging**